Assets/Scripts/Interactable/ShipFastTravel.cs

```diff
@@ -21,28 +21,27 @@ public class CheckFastTravel : MonoBehaviour
     public GameObject MainShip;
     public GameObject ShipParent;
 
-    public GameObject[] PlayersInScene;
-
     #endregion
 
     #region Private Variables
 
     private int _count = 0;
 
-    private FastTravelUI _travelDestination;
+    private FastTravelNPC _travelDestination;
     private ShipDoor _doorState;
-    private FastTravelUI _wantToTravel;
+    private FastTravelNPC _wantToTravel;
+    private PlayerInputHandler[] _playersInScene;
 
     #endregion
 
     #region Unity Loops
 
     private void Start()
     {
-        PlayersInScene = GameObject.FindGameObjectsWithTag("Player");
-        _travelDestination = FindObjectOfType<FastTravelUI>();
+        _playersInScene = FindObjectsOfType<PlayerInputHandler>();
+        _travelDestination = FindObjectOfType<FastTravelNPC>();
```

Comment:
This can't be a `FindObjectOfType` since there will be multiple `FastTravelNPC`s in the scene and doing this will retrieve a random `FastTravelNPC`